hyperspy/io_plugins/bcf.py

```diff
@@ -516,8 +576,7 @@ def __init__(self, xml_str):
         self._set_image(root)
         self.elements = {}
         self._set_elements(root)
-        self.line_counter = np.fromstring(str(root.LineCounter),
-                                          dtype=np.uint16, sep=',')
+        self.line_counter = interpret(root.LineCounter.text)
```

Comment:
1. it is quite short and numpy is not effective at creating short arrays
2. if it would be numpy array, then whole dict tree would be imposible to dump with json straight forwardly